app/controllers/katello/concerns/api/v2/exports_controller.rb

```diff
@@ -0,0 +1,172 @@
+module Katello
+  module Concerns
+    module Api::V2::ExportsController
+      extend ActiveSupport::Concern
+
+      class_methods do
+        def export_format_param
+          export_format_description = N_("Export formats."\
+                               "Choose syncable if the exported content needs to be in a yum format. "\
+                               "This option is only available for %{syncable_repos} repositories. "\
+                               "Choose importable if the disconnected importing server uses the same version "\
```

Comment:
Does the importing server need to be disconnected?

```suggestion
                               "Choose importable if the importing server uses the same version "\
```